frontend/express/public/core/user-management/javascripts/countly.views.js

```diff
@@ -9,9 +9,68 @@
             rows: Array
         },
         data: function() {
-            return {};
+            return {
+                tableFilter: null,
+                tableDynamicCols: [
+                    {
+                        label: "User",
+                        value: "full_name",
+                        default: true,
+                        required: true
+                    },
+                    {
+                        label: "Username",
+                        value: "username",
+                        default: true,
+                        required: true
+                    },
+                    {
+                        label: 'Role',
+                        value: 'global_admin',
+                        default: true,
+                        required: true
+                    },
+                    {
+                        value: "email",
+                        label: "E-mail",
+                        default: true,
+                        required: true
+                    },
+                    {
+                        value: "created_at",
+                        label: "Created at",
+                        default: true,
+                        required: true
+                    },
+                    {
+                        value: "last_login",
+                        label: "Last login",
+                        default: true,
+                        required: true
+                    }
+                ]
+            }
         },
-        created: function() {
+        computed: {
+            filteredRows: function() {
+                var self = this;
+                if (this.tableFilter) {
+                    return this.rows.filter(function(row) {
+                        if (self.tableFilter === 'global_admin') {
+                            return row.global_admin;
+                        }
+                        else if (self.tableFilter === 'admin') {
+                            return !row.global_admin && row.permission._.a.length > 0;
+                        }
+                        else {
+                            return !row.global_admin && row.permission._.a.length === 0;
```

Comment:
is the row object structured like -
```
{
permission: {
     _: {
         a: []
     }
   }
}
```

Reply:
yes, its configuration property for permission sets in drawer. :D but yes, its not great naming. Maybe I should consider rename that.